documentation/docfx_project_en/local/toc.yml

```diff
@@ -1,6 +1,6 @@
 - name: About Sample Code
   href: index.html#sample-code
-- name:  Steps to Execution
-  href: index.html#steps-to-execution
-- name: How to Customize and Use Autoware Packages
-  href: index.html#index.html#how-to-customize-and-use-autoware-packages
\ No newline at end of file
+- name:  Steps for Execution
+  href: index.html#steps-for-execution
+- name: Customizing Autoware
+  href: index.html#customizing-autoware
```

Comment:
```suggestion
  href: index.html#customizing-autoware

```

Reply:
Fixed in https://github.com/AutomotiveAIChallenge/aichallenge2023-sim/pull/25/commits/04fef7b7e599d808379e76a6186c86f164fbdf79